editors/code/src/commands.ts

```diff
@@ -947,10 +948,27 @@ export function openDocs(ctx: CtxInit): Cmd {
         const position = editor.selection.active;
         const textDocument = { uri: editor.document.uri.toString() };
 
-        const doclink = await client.sendRequest(ra.openDocs, { position, textDocument });
+        const doclinks = await client.sendRequest(ra.openDocs, { position, textDocument });
+
+        let doclink;
+        try {
+            if (
+                typeof doclinks.local === "string" &&
+                (await vscode.workspace.fs.stat(vscode.Uri.parse(doclinks.local)))
+            ) {
+                // file does exist locally
+                doclink = doclinks.local;
+            } else {
+                doclink = doclinks.web;
+            }
+        } catch (e) {
+            log.info("stat() threw error. Falling back to web version", e);
```

Comment:
This log isn't too useful (nor is the message in it), so we can drop this I think.

Reply:
My thougt process was it might be useful if `stat()` doesn't throw because of the documentation file being missing, but because of some different reason (corrupt filesystem, hardware fault, missing permissions, etc.). However, I do agree that this might be a bit chatty. I can remove it later today or tomorrow (don't have access to a computer at the moment).

Reply:
I'm fine with downgrading it `debug` in that case as well. (The chatty-ness was my concern here as well)

Reply:
Done.